packages/headless/src/features/search-parameters/search-parameter-analytics-actions.test.ts

```diff
@@ -53,51 +53,65 @@ describe('logParametersChange', () => {
     );
   });
 
-  it('should log #logFacetSelect when an #f parameter is added', () => {
+  testFacetLogging('f', expectIdenticalActionType);
+
+  testFacetLogging('af', expectIdenticalActionType);
+
+  testFacetLogging('cf', expectIdenticalActionType);
+
+  it('should log a generic #logInterfaceChange when an unmanaged parameter', () => {
     expectIdenticalActionType(
-      logParametersChange({}, {f: {author: ['Cervantes']}}),
+      logParametersChange({}, {cq: 'hello'}),
+      logInterfaceChange()
+    );
+  });
+});
+
+function testFacetLogging(
+  parameter: string,
+  expectIdenticalActionType: (
+    action1: SearchAction,
+    action2: SearchAction
+  ) => void
+) {
```

Comment:
In this file, the logging of 'cf' facets was not tested so I put the 'f' tests in a function to then be able to call it with different parameters.

Reply:
mhh do we wrap `it` statements in a function somewhere else in the code?
If so sure let's keep it that way. If not, maybe try describe.each?
https://jestjs.io/docs/api#describeeachtablename-fn-timeout

Reply:
Yes we do it elsewhere in the code. I think it is easier to read this way than wrapping inside a describeEach